### ISSUE TITLE
[BUG] Repeater's "Add new set" stops working

### ISSUE THREAD

**Issue by eduardomart** [bolt/bolt#6306]
I have a Content Type with 3 Repeaters, the first one, in the Content Tab seems to work fine, even if I change its tab. The other two don't seem to respond to the "Add new set" button though, and the following appears in the console:

![screen shot 2017-01-26 at 4 54 36 pm](https://cloud.githubusercontent.com/assets/1833020/22356754/8da57642-e3e8-11e6-8775-1afa0e96336b.png)

This is currently my full Content Type:
```
Productions:
  name: Productions
  singular_name: Production
  fields:
    title:
      type: text
      class: large
      group: content
    slug:
      type: slug
      uses: title
    composer:
      type: text
      label: Composer(s)
    image:
      label: Cover Photo/Image
      type: image
    repcompanies:
      label: Companies
      type: repeater
      fields:
        year:
          type: text
          label: Year
          variant: in
    repreviews:
      type: repeater
      label: Reviews
      group: Reviews
      fields:
        quote:
          label: Review
          type: markdown
        author:
          label: Author
          type: text
        publication:
          label: Publication
          type: text
    repcast:
      label: Artists & Cast
      type: repeater
      group: Artists
      fields:
        artist:
          label: Artist Name
          type: text
        role:
          label: Role(s)
          type: text
```

Details
-------

 - Relevant Bolt Version: 3.2.6 - 3.2.11
 - Install type: tarball install
 - PHP version: 5.6, 7.0
 - Used web server: Apache

**Comment by afroald** [comment]
This seems to happen when I put a field with type `markdown` in a repeater.

**Comment by SahAssar** [comment]
@eduardomart @afroald So this only happens when there is a markdown field in one of the repeaters, right? Getting a confirmation on that helps narrowing the issue down.

**Comment by eduardomart** [comment]
@SahAssar correct, MD in Repeaters seems to break the Repeaters themselves. I just tested in `3.2.11`. 

Demo:
![2017-04-26 13-44-23](https://cloud.githubusercontent.com/assets/1833020/25456216/13a63b3e-2a87-11e7-9512-3084eaea3f86.gif)

**Comment by SahAssar** [comment]
@eduardomart Does it break with a single repeater with a MD field for you too?

**Comment by eduardomart** [comment]
@SahAssar No, I just modified my CT to only have one Repeater with a MD field and in that case it _does not_ break.

**Comment by SahAssar** [comment]
Alright seems like I didn't test #5489 properly then... I only did it with one repeater.

**Comment by znegva** [comment]
I stumbled on the same problem and did some tests, it seems the position of the repeater-field is relevant.

working:
```
tests:
    name: Tests
    singular_name: Test
    fields:
        title:
            type: text
        foobar:
            type: text
        sections:
            type: repeater
            fields:
                fieldcontent:
                    type: markdown
```

NOT working:
```
tests:
    name: Tests
    singular_name: Test
    fields:
        title:
            type: text
        sections:
            type: repeater
            fields:
                fieldcontent:
                    type: markdown
        foobar:
            type: text
```


tested on Bolt 3.2.13

**Comment by bobdenotter** [comment]
Thanks for the test-case, @znegva! A fix for this is in the PR queue at #6719
